pep/app/pods/components/document/read/component.ts

```diff
@@ -52,13 +52,13 @@ export default class DocumentRead extends Component<DocumentReadArgs> {
      * @memberof ReadDocument
      */
     @action
-    viewSearch(searchTerms: string) {
+    viewSearch(search: object) {
```

Comment:
Any chance we could Type this object? I assume not since you probably would've thought of that 😛